storage/store.go

```diff
@@ -1269,10 +1269,61 @@ func (s *Store) ReplicaCount() int {
 // ExecuteCmd fetches a range based on the header's replica, assembles
 // method, args & reply into a Raft Cmd struct and executes the
 // command using the fetched range.
-func (s *Store) ExecuteCmd(ctx context.Context, args proto.Request) (proto.Response, error) {
+func (s *Store) ExecuteCmd(ctx context.Context, args proto.Request) (reply proto.Response, _ error) {
 	ctx = s.Context(ctx)
 	trace := tracer.FromCtx(ctx)
-	// If the request has a zero timestamp, initialize to this node's clock.
+	if trace == nil {
```

Comment:
to get test output for cases where the request is fed straight into this function, without a trace in the context.